### DISCUSSION TITLE
贏家思維

### DISCUSSION THREAD

**Discussion by harrycoconut** [1101-GS4719-C/Python-Community-GS4719-C#1292] [General]
最近看了一本書叫做贏家思維，作者是科比布萊恩跟麥可喬丹的傳奇訓練師，在接觸過這些超強的運動員後，他發現這些贏家都有幾個獨特的觀點，而其中最令我印象深刻的就是作者形容勝利的形象。
勝利是邪惡的，他會在你以為能抓住他的時候推你一把，讓你墜入谷底；勝利是無情的，當你以為抓住他時，下一秒他已悄然離去，只剩空虛的你留在原地。
這樣的形容是我第一次聽過，同時也非常認同😌許多書都一直強調勝利是多麼美好，成功之後便能高枕無憂，然而現實中的勝利卻十分無情，你必須付出你的一切，show everything you got，就算如此也不一定能抓住勝利👁️👄👁️

**Comment by Chienyuuu6** [comment]
希望你在看這本書的時候有想到TEDxNCU Salon帶給你的ideas～

我最近對於「贏家思維」也有一些見解：我認為處在這個環境下的人們常常被「量化」標準給綁架，不管是學業成績高低、身形體重胖瘦等等的，這些標準都是「統計」下來社會覺得「優秀」的代表，達到一定標準就是個贏家。

然而很多沒有辦法被量化的事情其實不代表不好，前陣子有朋友對名次排名綁架了，我鼓勵他「知道自己努力了多少付出了多少」就好了，自己給予「自己肯定」與價值比其他人給的來得更有意義！

**Comment by 1104030360** [comment]
非常有趣的分享，我常常在網路上看到許多成功人士的文章，就會認為成功是十分容易取得的，卻忽略了要獲得成功過程需要多麼的辛苦，這提醒了我把握住成功是十分不容易的，希望你在各方面領域都能保持這個心態😍！

**Comment by KevinCheng42** [comment]
許多人都想獲得勝利 ，都想成功，但是大部分人都覺得只要照著成功人士的步伐便可得到一樣的結果。事實上，「倖存者偏差」普遍存在於我們的社會中，我覺得我們必須謹記成功是不易達成的，並且需要付出大量心力～

**Reply by HsuSungTing** [reply]
真的很同意，當別人在慶祝自己的成功時，我們不知道別人實際付出有多少
而且回到自己身上時，如果長期要求自己在各方面都維持完美，壓力真的會很大，而且會很難接受自己的任何失誤。
我認為，我們不該全盤接受他人所定義的成功或標準，應該跟這些標準保持是當的距離，以及從肯定自己的付出開始。

**Comment by CCello** [comment]
這本書的作者有一個獨特的觀點，他看待勝利的角度不同於其他書籍，他認為勝利並非一件容易得到的事情，而是需要付出很多努力和代價的，也可能是一個永遠無法到達的目標。這種觀點提醒我們，在追求勝利的過程中，必須要有毅力和耐力，並且要準備好面對失敗和挫折。

不過，我們也不能因此就放棄追求勝利，因為勝利本身也是一個有價值的目標，它可以讓我們更好地認識自己，挑戰自己的極限，獲得成就感和自信心。因此，我們應該學會平衡追求勝利和面對失敗的能力，保持一個積極向上的態度，並不斷學習和進步。

**Comment by tower20021202** [comment]
這本書的觀點我也很認同，往往在往一個方面努力時，需要隨時做好面段失敗和挫折，即使你已經奮不顧身地不斷前進，但當我們突破一個障礙後，往後依然會出現更高的難題來阻礙我們，但正是因為這樣，我們才能在一次次的突破中成為更好的自己，更加接近我們所想要的成果

**Comment by Jack-1D** [comment]
就像我聽許多人描述過自己的成功是個巧合，然而機會是留給準備好的人。即便只是運氣好，但如果想要成功，我們還是得要時時準備好，並在機會來臨時，牢牢抓緊機會。

**Comment by YoYooHuang** [comment]
勝負充斥在我們的生活中，不論考試，球賽等等，大家都以名次來衡量一切。但我也不覺得這是壞事，反而能因此更加進步，審視自己的不足，了解與別人的差距，做出相應的改變。但還是有些人會因為一次失敗而一蹶不振，期待大家都能更加堅強，從失敗中學習。

**Comment by zilinng** [comment]
我覺得要取得勝利必須天時地利人和，而其中只有個人能力是我們有辦法把握在自己手上的，其他就只能靠運氣，雖然聽起來感覺人的努力真的很渺小，就算做到要死要活也不一定能成功，但如果連自己都不努力的話成功的機率就更渺茫了，所以我們只能盡力提升自己的能力，其他就放寬心，不要對勝負太執著，才能過的比較開心

**Comment by ziihui** [comment]
我認為勝利是需要付出一定代價的，而且它並不總是如我們想像中的那樣美好😢在現實中，我們必須付出很多努力，甚至可能會有失敗的經驗。但正是這些經驗，讓我們能夠成為更好的人，更接近真正的勝利！💪一起加油🫡

**Comment by Evening218** [comment]
我最近看的書裡也有提到關於勝利或是成功的定義,勝利並不一定是要和別人比較,或者是要做得比別人好,更適合的定義應該是比之前的自己做得更好,無論是堅持一件事的時間變長,或著是完成一件曾經做不到的事,只要每一秒都比上一秒的自己更加優秀就是勝利

**Comment by llh26** [comment]
我覺得其實我們很常只看到人們成功、光鮮亮麗的那一面，但往往忽略了他們背後可能付出的努力和代價，就和書中作者提到的想法相呼應，勝利的本質是邪惡的，並非我們想像中的那麼美好，有付出才會有回報！之前聽過長輩分享，其實那些人成功後，並不能永遠快樂或享受，成功的背後可能有更多事情要去煩惱，有得必有失。
